### PR TITLE
LIT6470 71 72 80 81

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New entries. One Malke'a Mamas is too much, I will delete it in a different PR. A correction in the record for Agerhese Abana Mamas is necessary, I will do it.

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! Just for the future, could you please provide in the initial comment of the PR the link to the issue where these works were discussed?

File: new/LIT6481DossierMamas.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6481DossierMamas" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Hagiographic Dossier of Māmās</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT6481DossierMamas.xml:8]: ok, i will do

File: new/LIT6480MalkeaMamas.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6480MalkeaMamas" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">MalkeaMamas</title>
```
Comment (eu-genia) [inline new/LIT6480MalkeaMamas.xml:8]: transcription to correct

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
Just to link here: https://github.com/BetaMasaheft/Documentation/issues/1773

**Comment by eu-genia** [comment]
still corrections to do? 
by the way LIT4775GadlaMa is about still another Mamas (PRS6626Mammas)? 
(I guess we also have to harmonize Mamas/Mammas)

**Comment by eu-genia** [comment]
so PRS6626Mammas is the same as PRS13452Mamas?
and LIT4775GadlaMa is the same as LIT6470GadlaMamas (not yet merged)?

**Comment by DenisNosnitsin1970** [comment]
Yes. "Gadla Māmmās za-Gʷǝgʷǝdā" LIT4775GadlaMa is the same.

**Comment by eu-genia** [comment]
So I suggest that the information from LIT4775GadlaMa is moved to LIT6470GadlaMamas (title, KRZ) and 4775 can be deleted (wuth a relation formerlyListedAs)
Also PRS6626Mammas and PRS13452Mamas should be merged

**Comment by DenisNosnitsin1970** [comment]
OK.

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Comment by thea-m** [comment]
Now the deletion of LIT6480MalkeaMamas remains for a new PR

**Comment by DenisNosnitsin1970** [comment]
indeed.

**Pull request merged**